### PR TITLE
Improvement/batch ui fixes

### PR DESCRIPTION
**Pull request by szymon-owczarzak**
Remove option to expand IFrame widget
Checkbox widget now have it own set of statuses
Text widget now can be expanded
Expanded widget casts shadow
Settings button moved next to User login button
Components styles cleanup
Selected tab indication fixed
No boards text is now visible - fixed background
Error page style fix
DefaultWidget changed to WhiteSpaceWidget

### CODE REVIEW

File: cogboard-webapp/src/components/Widget/index.js [hunk 1]
```diff
@@ -121,6 +121,8 @@ const Widget = ({ id, index }) => {
   return (
     <>
       <StyledCard
+        showShadow={expanded}
+        showBorder={isAuthenticated && 'WhiteSpaceWidget' === type}
```
Comment (Southeaven) [inline cogboard-webapp/src/components/Widget/index.js]: Wouldn't it be better to move `isAuthenticated && 'WhiteSpaceWidget' === type` to  variable and use it throughout component?

File: docs/widget-development.md [hunk 2]
```diff
@@ -9,37 +9,41 @@ All available widgets are listed in `cogboard-webapp/src/components/widgets/type
 **The very basic structure of the widget component looks as follows:**
 
 ```javascript
-import React from 'react';
+import React from "react";
 
-import Typography from '@material-ui/core/Typography';
+import Typography from "@material-ui/core/Typography";
```
[the remark this answers is not in the document]
Comment (szymon-owczarzak) [reply docs/widget-development.md]: no this is some formatter fix - i will revert this

File: cogboard-webapp/src/components/Widget/index.js (hunk 1 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/Widget/index.js]: done

Comment (goeson00) [inline cogboard-webapp/src/components/Widget/index.js]: Move this logic to const and type === 'WhiteSpaceWidget' seems more natural for me.

File: cogboard-webapp/src/components/Widget/index.js [hunk 3]
```diff
@@ -145,7 +147,13 @@ const Widget = ({ id, index }) => {
               color: 'textPrimary'
             }}
             action={
-              <MoreMenu>
+              <MoreMenu
+                color={
```
Comment (goeson00) [inline cogboard-webapp/src/components/Widget/index.js]: The sam with this.

File: cogboard-webapp/src/components/Widget/index.js (hunk 1 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/Widget/index.js]: done

File: cogboard-webapp/src/components/Widget/index.js (hunk 3 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/Widget/index.js]: done

**Pull request merged**